backend/roundrobin/RoundRobinRegisterApi/README.md

```diff
@@ -0,0 +1,11 @@
+## Round Robin Register API
+
+this is the backend of Round robins, it uses API gateway, AWS lambda, Java 21 and maven
+
+## Build
+
+- `mvn clean`
+- `mvn package`
+- `cd target`
+- deploy RoundRobinRegisterAPI-1.0-SNAPSHOT.jar to lambda

```

Comment:
Can you check how to use Java with serverless? I'd greatly prefer to not have manual steps in our deployment process